docs/examples.md

```diff
@@ -270,6 +270,54 @@ Will return GeoJSON:
 }
 ```
 
+## Get time series GeoJSON for a lake
+
+Search for a single lake by ID.
+
+[https://soto.podaac.earthdatacloud.nasa.gov/hydrocron/v1/timeseries?feature=PriorLake&feature_id=6350036102&start_time=2024-07-20T00:00:00Z&end_time=2024-07-26T00:00:00Z&fields=lake_id,time_str,wse,area_total,quality_f,collection_shortname,crid,PLD_version,range_start_time&output=geojson](https://soto.podaac.earthdatacloud.nasa.gov/hydrocron/v1/timeseries?feature=PriorLake&feature_id=6350036102&start_time=2024-07-20T00:00:00Z&end_time=2024-07-26T00:00:00Z&fields=lake_id,time_str,wse,area_total,quality_f,collection_shortname,crid,PLD_version,range_start_time&output=geojson)
+
+Will return GeoJSON:
```

Comment:
let's add a note here about how due to the size of the original polygon, in this version we are only returning the calculated center point of the lake. This is to facilitate conformance with the geojson spec and centerpoints should not be considered accurate.

Reply:
Great idea! I updated the `examples.md` page as well as the `overview.md` to include this as a limitation alongside the response size. I also added the lake dataset to the `intro.md` page.